### PR TITLE
fix: can't enter VR session with WebXR

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This fixes the issue that the user can't enter VR mode with the latest WebXR API

### CODE REVIEW

File: demo/common/js/PanoControls.js [hunk 1]
```diff
@@ -24,14 +24,15 @@ var PanoControls = (function() {
 		var pieView = new PieView(pieEl);
 		panoViewer.on({
 			"ready": function () {
-				var yawRange = panoViewer.getYawRange();
-				pieView.setState(panoViewer.getYaw(), panoViewer._getHFov(), yawRange[1] - yawRange[0]);
+        var yawRange = panoViewer.getYawRange();
+        var hfov = (180 / Math.PI) * (2 * Math.atan(panoViewer._aspectRatio * Math.tan(panoViewer.getFov() * Math.PI / 360)));
```
Comment (mixed) [inline demo/common/js/PanoControls.js:28]: How about subtracting with a function even though it is the same sentence?

Comment (mixed) [reply demo/common/js/PanoControls.js:28]: Ah... It's a demo. There is no big issue even if you don't change it. But, it's a bit weird to access internal variables in a class.

File: test/manual/PanoViewer/Panorama.html [hunk 2]
```diff
@@ -68,25 +68,26 @@ <h2>Select your panorama picture taken on smartphone</h2>
...
         window.addEventListener("resize", debounce(function() {
-            panoViewer.updateViewportDimensions();
-            
-            var yaw = panoViewer.getYaw();
-            var hfov = panoViewer._getHFov();
-            pieView.setState(yaw,  hfov);
+          panoViewer.updateViewportDimensions();
+
+          var yaw = panoViewer.getYaw();
+          var hfov = (180 / Math.PI) * (2 * Math.atan(panoViewer._aspectRatio * Math.tan(panoViewer.getFov() * Math.PI / 360)));
```
Comment (mixed) [inline test/manual/PanoViewer/Panorama.html:89]: same issue.

**Review by mixed — APPROVED** [review]
It's a demo, but accessing internal variables is a bit weird.

File: demo/common/js/PanoControls.js (hunk 1 above)
Comment (WoodNeck) [reply demo/common/js/PanoControls.js:28]: This PR is trying to resolve a previous issue that removing the unnessesary `_getHFov` method breaks the View360 demo.
I won't add additional methods like `getAspectRatio` in this PR as it'll end up increase our minor version.
In other words, I'd like this PR to be just a hotfix. That should be resolved in another PR I think.

**Review by daybrush — APPROVED** [review]
LGTM

**Comment by tranhogdiep** [comment]
Hello, any update on it?
Thank you.

**Pull request merged**